lib/src/handle_access_token_response.dart

```diff
@@ -143,14 +126,4 @@ void _handleErrorResponse(http.Response response, Uri tokenEndpoint) {
   var uriString = parameters['error_uri'];
   var uri = uriString == null ? null : Uri.parse(uriString);
   throw new AuthorizationException(parameters['error'], description, uri);
-}
-
-void _validate(
-    http.Response response,
-    Uri tokenEndpoint,
-    bool condition,
-    String message) {
-  if (condition) return;
-  throw new FormatException('Invalid OAuth response for "$tokenEndpoint": '
-      '$message.\n\n${response.body}');
-}
```

Comment:
This still needs to exist; right now, validation outside of `getParameters()` won't include the token endpoint or the response body.